bin/build-harmony

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# build the cmr granule locator task image
+# cd tasks/query-cmr
+# docker build --tag query-cmr:latest .
+# cd ../..
+
+# build the harmony-local docker image
+docker build --tag harmony/harmony-local:latest .
+
+# build the cmr-granule-locator image
+docker build --tag harmony/query-cmr:latest -f ./Dockerfile.cmr-granule-locator-k8s .
```

Comment:
After my PR, this needs to get changed to `harmonyservices/query-cmr`

Reply:
ok

Reply:
I have deleted this line - no need to build the Dockerfile here if it is in DockerHub and you presumably have a way to build it somewhere else.